### DISCUSSION TITLE
Creating an API key for the first fief user automatically

### DISCUSSION THREAD

**Discussion by ricardogsilva** [fief-dev/fief#125] [Ideas]
It would be great if the mechanism that already exists for creating the first fief user would also allow creating a default API key for administrating fief.

I imagine that in addition to the `FIEF_MAIN_USER_EMAIL` and `FIEF_MAIN_USER_PASSWORD` there could be another env variable (perhaps `FIEF_MAIN_USER_API_KEY`?) that would take care of creating a default API key for the main user.

This would make it possible to streamline bootstrapping operations - my concrete use case is to add custom user fields.

**Comment by fief-bailiff** [comment]
Hail, @ricardogsilva 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
Hello @ricardogsilva 👋 

That's indeed an interesting idea! I'll add it to the backlog 👍
